src/RedHat.AspNetCore.Server.Kestrel.Transport.Linux/TransportThread.TSocket.cs

```diff
@@ -859,10 +859,10 @@ public unsafe PosixResult TrySetSocketOption(SocketOptionLevel optionLevel, Sock
                 return SocketInterop.SetSockOpt(Fd, optionLevel, optionName, (byte*)&value, 4);
             }
 
-            public unsafe void Bind(IPEndPointStruct endpoint)
+            public unsafe PosixResult  TryBind(IPEndPointStruct endpoint)
```

Comment:
nit: remove double space
```suggestion
            public unsafe PosixResult TryBind(IPEndPointStruct endpoint)
```